cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -409,34 +412,40 @@ private void configureAuthentication(String listenerName, List<String> securityP
         if (auth instanceof KafkaListenerAuthenticationOAuth oauth) {
             securityProtocol.add(String.format("%s:%s", listenerName, getSecurityProtocol(tls, true)));
 
-            List<String> options = new ArrayList<>(getOAuthOptions(oauth));
-            options.add("oauth.config.id=\"" + listenerName + "\"");
+            Map<String, String> jaasOptions = new LinkedHashMap<>(getOAuthOptions(oauth));
```

Comment:
Just curious, why do you use `LinkedHashMap` instead of HashMap? :) I do not see anything related to the order so using HashMap should be sufficient or am I missing something?.

Reply:
because the tests in the KafkaBrokerConfigurationBuilderTest.java is failing when we have HashMap instead of LinkedHashMap and that is because the order matter in this situation